app/screens/home/search/results/header.tsx

```diff
@@ -131,11 +136,9 @@ const Header = ({
                     onPress={handleFilesPress}
                     text={`${filesText} (${numberFiles})`}
                 />
-                <View
-                    style={styles.filter}
-                >
+                <View style={styles.filter}>
```

Comment:
Nit: is it still meaningful this view to be named filter? Since it is the container for both the team and the filter, should it have a different name? 0/5